tests/global.rs

```diff
@@ -134,3 +138,87 @@ fn hammer_split() {
         assert_eq!(N * 4, cnt.load(Relaxed));
     }
 }
+
+#[test]
+fn spawn_handle() {
+    let f = lazy(|| {
+        Ok::<_, ()>("hello from the future")
+    });
+
+    let res = tokio::run(lazy(move || {
+        tokio::spawn_handle(f)
+            .then(|res| {
+                assert_eq!(res.unwrap(), "hello from the future");
```

Comment:
How does the test ensure that this line gets run without any panics? the threadpool *definitely* catches panics.